checkov/terraform/checks/resource/aws/DBInstanceLogging.py

```diff
@@ -17,8 +17,10 @@ def get_inspected_key(self) -> str:
         return "enabled_cloudwatch_logs_exports/[0]"
 
     def scan_resource_conf(self, conf: Dict[str, List[Any]]) -> CheckResult:
-        logs_exports = conf.get('enabled_cloudwatch_logs_exports', [[]])[0]
-        return CheckResult.PASSED if logs_exports else CheckResult.FAILED
+        logs_exports = conf.get('enabled_cloudwatch_logs_exports', [[]])
+        if len(logs_exports) == 0:
```

Comment:
```suggestion
        if not logs_exports:
```
does the same, but faster 🙂